util/profiledata.js

```diff
@@ -85,7 +86,7 @@ module.exports = async function (args, context = epochtal) {
 
       if (profilesPath) {
         const dataPath = `${profilesPath}/${steamid}/data.json`;
-        await Bun.write(dataPath, JSON.stringify(profile));
+        if (fs.existsSync(dataPath)) await Bun.delete(dataPath);
```

Comment:
are we... sure this is how you delete a file with bun? i thought there was no bun api for that:
https://bun.sh/guides/write-file/unlink